spec/suites/map/MapSpec.js

```diff
@@ -1318,4 +1318,14 @@ describe("Map", function () {
 			expect(map.getZoom()).to.be(undefined);
 		});
 	});
+
+	describe("#Geolocation", function () {
+		it("don't throw error if location is found and map is not existing", function () {
+			var fn = L.Util.bind(map._handleGeolocationResponse, map);
```

Comment:
Perhaps you should ass location error test as well

Reply:
Have you an idea how to test? I don't throw an error so I can't catch them

Reply:
Maybe I can listen if the `locationerror` event is fired

Reply:
> Have you an idea how to test?

In the same manner - call `_handleGeolocationError` and check it does not throw.

Reply:
`_handleGeolocationError` doesn't throw an error, so I can't catch it

Reply:
Even if `setView` option is used?

Reply:
You are right. I have tested it wrong (while map initialisation I set center and zoom) and then setView was not called, because the map was already loaded.